services/wfm/src/main/java/org/openkilda/wfm/topology/event/OfEventWfmTopology.java

```diff
@@ -95,10 +95,23 @@ public StormTopology createTopology() throws StreamNameCollisionException {
         BoltDeclarer bd = builder.setBolt(DISCO_BOLT_ID, ofeLinkBolt, topologyConfig.getParallelism())
                 .shuffleGrouping(DISCO_SPOUT_ID);
 
+        PersistenceManager persistenceManager =  PersistenceProvider.getInstance()
+                .createPersistenceManager(configurationProvider);
+
+        NetworkTopologyBolt networkTopologyBolt = new NetworkTopologyBolt(persistenceManager,
+                (int) topologyConfig.getIslCostWhenPortDown());
```

Comment:
We should change type of second argument of `NetworkTopologyBolt` or change type returned from `topologyConfig.getIslCostWhenPortDown()` ... in other words they should match

(minor)